### PR TITLE
Onboarding chrisproject

### PR DESCRIPTION
**Pull request by shreekarSS**
For issue https://github.com/operate-first/support/issues/581
Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

File: cluster-scope/base/core/namespaces/chrisproject-dev/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: chrisproject-dev
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/limitranges/default
+    - ../../../../components/project-admin-rolebindings/chris-project
```
Comment (harshad16) [inline cluster-scope/base/core/namespaces/chrisproject-dev/kustomization.yaml:8]:
```suggestion
    - ../../../../components/project-admin-rolebindings/chris-project
    - ../../../../components/resourcequotas/small
```

include the quota please

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
some suggested changes

File: cluster-scope/base/core/namespaces/chrisproject-dev/kustomization.yaml (hunk 1 above)
Comment (shreekarSS) [reply cluster-scope/base/core/namespaces/chrisproject-dev/kustomization.yaml:8]: Ack

**Comment by Gregory-Pereira** [comment]
Before the onboard command can be properly used we need to change our standard format for the two overlay kustomizations (cluster-scope/overlays/prod/common/kustomization.yaml and cluster-scope/overlays/prod/moc/smaug/kustomization.yaml). Because of this, could you keep the current operate-first/apps kustomization format. This should reduce the diff in the PR so we only see the user groups and namespace added to those two kustomization files. Let me know if you have any questions about this.

**Comment by shreekarSS** [comment]
> Before the onboard command can be properly used we need to change our standard format for the two overlay kustomizations (cluster-scope/overlays/prod/common/kustomization.yaml and cluster-scope/overlays/prod/moc/smaug/kustomization.yaml). Because of this, could you keep the current operate-first/apps kustomization format consistent with what master currently reflects? This should reduce the diff in the PR so we only see the user groups and namespace added to those two kustomization files. Let me know if you have any questions about this.


Thanks , fixed the indentation. Format is same as operate-first/apps masters branch.

**Review by harshad16 — COMMENTED** [review]
/lgtm
thanks :+1:

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm

**Review by harshad16 — APPROVED** [review]
/lgtm

**Pull request merged**